core/src/components/nav/nav.tsx

```diff
@@ -607,25 +579,33 @@ export class Nav implements NavOutlet {
     }
   }
 
+  /**
+   * Consumes the next transition in the queue.
+   *
+   * @returns whether the transition is executed.
+   */
   private nextTrns(): boolean {
     // this is the framework's bread 'n butta function
     // only one transition is allowed at any given time
     if (this.isTransitioning) {
       return false;
     }
 
-    // there is no transition happening right now
-    // get the next instruction
+    // there is no transition happening right now, executes the next instructions.
     const ti = this.transInstr.shift();
     if (!ti) {
       return false;
     }
-
-    this.runTransition(ti);
+    this.runTransitions(ti);
     return true;
   }
 
-  private async runTransition(ti: TransitionInstruction) {
+  /**
+   * Executes all the transition instruction from the queue.
+   *
+   * @param ti The transition instructions.
```

Comment:
Can we make this more descriptive or remove altogether? There isn't any information in here that I could not already deduce from the type name.

Reply:
Removed